executor/common_linux.h

```diff
@@ -3785,6 +3785,7 @@ static int do_sandbox_none(void)
 	if (pid != 0)
 		return wait_for_loop(pid);
 
+	setup_binderfs();
```

Comment:
Note that here (and in other places) `setup_binderfs()` was called after `setup_common()` because we do `unshare(CLONE_NEWNS)` there.

There's a possibility that it would still grant each syz-executor process its own binderfs mount (we mount `/dev` as `MS_PRIVATE` --> further /dev/binderfs mounts should not propagate to existing processes --> syz-executor will only see the one that was in place during `unshare`). But this looks potentially racy and very fragile.

I think it would be safer to just move `setup_binderfs()` before seccomp filter configuration in `do_sandbox_android()`.

Reply:
Done